DEV-1316 Add subtitle request to sequence diagram

WardMaes opened the pull request:
Update UML diagram; a `makeSubtitleAvailableRequest` is sent. 

![image](https://user-images.githubusercontent.com/10989657/116065800-ac831680-a687-11eb-861e-8bf526170602.png)

WardMaes committed:
DEV-1316 Add subtitle request to sequence diagram

Update UML diagram; a `makeSubtitleAvailableRequest` is sent.

maartends reviewed (commented):
Ideally, and more correctly, this would be wrapped in een `opt` block.

WardMaes committed:
DEV-1316 Make subtitle request optional

The subtitle request is only sent when `media_type` is `video` and when either `metadata.closedOT_available` or `metadata.openOT_available` is set on the event.

WardMaes commented:
Update: 
![image](https://user-images.githubusercontent.com/10989657/116202532-f842c800-a73a-11eb-9548-f93207920450.png)
